icons.md

```diff
@@ -21,11 +21,35 @@ layers:
     my-layer:
         data: { source: my-source }
         draw:
-            icons:
+            mapzen_icon_library:
                 size: [[13, 18px], [16, 18px], [18, 22px]]
                 sprite: zoo
 ```
 
+## Using Mapzen icons independently
+
+You can use the icons in this library independent of one of the Mapzen basemap styles, but there are a few required steps to doing this in Tangram:
```

Comment:
To:

`You can use the icons in the Mapzen icon library independent of one of the Mapzen basemap styles by importing just the icon assets.`